bundle/Resources/views/ContentType/field_types.html.twig

```diff
@@ -116,6 +116,27 @@
     </div>
 {% endblock %}
 
+{% block ezobjectrelationlist_field_definition_edit %}
+    <div class="ezobjectrelationlist-settings selection-default-location">
+        {{- form_label(form.selectionDefaultLocation) -}}
+        {{- form_errors(form.selectionDefaultLocation) -}}
+        {{- form_widget(form.selectionDefaultLocation) -}}
+
+        <button data-universaldiscovery-title="{{"field_definition.ezobjectrelationlist.selection_root_udw_title"|trans({}, "ezrepoforms_content_type")}}"
+                class="ez-button-tree pure-button ez-font-icon ez-button ez-relation-pick-root-button"
+                data-relation-root-input-selector="#{{form.selectionDefaultLocation.vars.id}}"
+                data-relation-selected-root-name-selector="#selected_item_name"
+                >{{"field_definition.ezobjectrelationlist.selection_root_udw_button"|trans({}, "ezrepoforms_content_type")}}</button>
+        <div id="selected_item_name"></div>
```

Comment:
I suppose that `id` attribute should be generated dynamically as there may be many fields that type in a contenttype. Then this ID should be passed to `data-relation-selected-root-name-selector` above.
What's more this `<div></div>` should as default contain the name of already selected content.